src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -175,6 +176,13 @@ function ProductDetails({ product: staleProduct }: Props) {
               Add to Cart
             </BuyButton>
           )}
+          {!availability && (
+            <OutOfStock
+              onSubmit={() => {
+                console.info('test')
```

Comment:
Minor suggestion just to reiterate that the passed function is called with the email so this information is available at this point in time:

```suggestion
              onSubmit={(email) => {
                console.info(email)
```

Reply:
Perfect!